{{cookiecutter.project_shortname}}/scripts/bootstrap

```diff
@@ -6,38 +6,60 @@ set -e
 script_path=$(dirname "$0")
 pipfile_lock_path="$script_path/../Pipfile.lock"
 
+
+function check_for_pipfilelock {
+  if [ ! -f $pipfile_lock_path ]; then
+    if [ "$flags" == "--deploy" ]; then
+      echo "Pipfile.lock' not found. Exiting"
+      exit 1
+    else
+      echo "'Pipfile.lock' not found. Generating via 'pipenv lock --dev'..."
+      pipenv lock --dev
+    fi
+  fi
+}
+
+
 # The real bootstrap script starts 20 lines below ;-)
 flags=()
 while test $# -gt 0; do
     case "$1" in
         # If you need to add special flags to this script, this is the place:
-        # --ci)
-        #     flags=("--dev")
+         --ci)
+             flags=("--deploy")
         #     shift
-        #     ;;
+             ;;
         *)
             # Use the flags that are passed to the script directly to sync
             flags+=("$1")
             shift
             ;;
     esac
 done
+
+
+# If no flag was passed to boostrap script, set to dev env.
 if [ ${#flags[0]} -eq 0 ]; then
     flags=("--dev")
 fi
 
-
-if [ ! -f $pipfile_lock_path ]; then
-    echo "'Pipfile.lock' not found. Generating via 'pipenv lock --dev'..."
-    pipenv lock --dev
+if [ "$flags" == "--dev" ]; then
+  check_for_pipfilelock
+  pipenv sync --dev
+  # Install application code and entrypoints from 'setup.py'
+  pipenv run pip install -e $script_path/..
+  # Build assets
+  pipenv run invenio collect -v
+  pipenv run invenio webpack buildall
+elif [ "$flags" == "--deploy" ]; then
+  check_for_pipfilelock
+  pipenv install --system --deploy
+  # Install application code and entrypoints from 'setup.py'
```

Comment:
I don't think we should be using ``-e`` for production? (i know it's coming from the previous)